### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by micheledicosmo** [leafac/kill-the-newsletter#60]
Is there a dockerized version of this project? Is not, I might make one and contribute.

Do you have any heads up on things to be aware of before I start doing it?

**Comment by phampyk** [comment]
I found this https://github.com/leafac/kill-the-newsletter/tree/b9907a8021cebd6cd4a4b3131fd31ac5aa0a2e37 it has the dockerfile there

**Comment by micheledicosmo** [comment]
Thanks @phampyk. I also found issue #45 for history. I'll look into trying if any on these two approaches still work.

**Comment by leafac** [comment]
Hi @micheledicosmo & @phampyk 👋

This project doesn’t support Docker currently. I encourage you to use Kill the Newsletter! with Docker if you wish, and I appreciate the intent of contributing, but I’m not planning on maintaining a Docker configuration.

It should be relatively straightforward, too. Start with an image of a Linux distribution that supports glibc, for example, Ubuntu (I don’t think that Alpine would work), or maybe even Windows. Then grab the Kill the Newsletter! executable. Mount the `data/` directory as a volume. And it should be good to go.

My reasoning for not wanting to maintain a Docker configuration: Docker is yet another piece of infrastructure to learn about and manage, which adds on top of many moving parts that can go wrong. Particularly for beginner system administrators, this can be challenging. As @phampyk pointed out, in the past I supported Docker, but then I decided to instead make Kill the Newsletter! easier to install and manage. That’s why I set out to create https://github.com/leafac/caxa and started using SQLite as a database.

Thanks for reaching out 😁
